sw/animation/animation_thread.h

```diff
@@ -34,11 +35,14 @@ void main_loop_function()
 
   // Draw fixed one time objects
   static draw drawer; // Drawer object
+  bmp_loc = "conf/environments/"+ param->environment()+ "/gas_simulations/iteration_"+std::to_string((int)(floor(simtime_seconds)))+".bmp";
```

Comment:
At first glance this is too hard-coded for the animation thread, what if the user doesn't want to use gas simulations? What happens?